camino/docker-compose.dev.override.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.8"
+services:
+  nginx:
+    image: nginx
+    volumes:
+      - /etc/ssl/certs/dev.cep.tacc.utexas.edu.cer:/etc/ssl/certs/portal.cer
+      - /etc/ssl/private/dev.cep.tacc.utexas.edu.key:/etc/ssl/private/portal.key
```

Comment:
This PR is strictly for rebranding for this release, correct?

Reply:
Rebranding and preparing the repo for camino v2. However I think this stuff is no longer required, I'll remove it.

Reply:
> Rebranding and preparing the repo for camino v2. However I think this stuff is no longer required, I'll remove it.

👍  was just gonna suggest what you are already planning to do.  (i like that its all now in the other repo https://github.com/TACC/Core-Portal-Deployments/tree/main/core-portal/camino)